kinto/tests/test_views_collections_cache.py

```diff
@@ -0,0 +1,81 @@
+from .support import (BaseWebTest, unittest, MINIMALIST_RECORD)
+
+
+class ForcedCacheExpiresTest(BaseWebTest, unittest.TestCase):
```

Comment:
I don't understand what is the fundamental difference between this class and the latter one.